src/bottom-bar/bottom-bar.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react'
-import { useHighlightedFieldIdContext } from '../shared/index.js'
+import { useHighlightedFieldStore } from '../shared/index.js'
 import styles from './bottom-bar.module.css'
 import DataItemBar from './data-item-bar.js'
 import MainToolBar from './main-tool-bar.js'
 
 export default function BottomBar() {
-    const { item } = useHighlightedFieldIdContext()
+    const item = useHighlightedFieldStore((state) =>
```

Comment:
I feel like the choice of the word `item` here is now particularly confusing because you are only getting the highlighted field id at this point, which is less clear because `useHighlightedFieldStore` as a name makes it less clear than `useHighlightedFieldIdContext` that the store contains IDs and not the full item information (i.e. from use-highlighted-field). I'd suggest renaming here, or renaming the store

Reply:
Yeah I found naming this was particularly difficult. We refer to `${dataElementId}.${cocId}` as `fieldId`, and this store stores those ids in an object. I did initially just want to call this `field` or `highlightedField`- but it's not really descriptive either. Then I renamed the item in the store to `highlightedFieldId`, due to a comment above. But I'm not sure I like that either, since `id` have a strong connotation to the value being a string. So I think I would prefer `highlightedField` now, even if `field` doesn't really describe what it is.

Reply:
Yeah. I think `highlightedField` would be better 👍